tests/test_create_model.py

```diff
@@ -222,3 +222,27 @@ class TestGenericModel(GenericModel):
     result = AAModel[int](aa=1)
     assert result.aa == 1
     assert result.__config__.orm_mode is True
+
+
+def test_set_name():
+
+    from unittest.mock import Mock
```

Comment:
you don't really need this, you could just do

```py
calls = []
...
def __set_name__(self, owner, name):
    calls.append((owner, name))
```
But if you do want to use mock, please make it a global import to match the style of other tests.